ethers-etherscan/src/errors.rs

```diff
@@ -23,6 +23,8 @@ pub enum EtherscanError {
     Serde(#[from] serde_json::Error),
     #[error("Contract source code not verified: {0}")]
     ContractCodeNotVerified(Address),
+    #[error("Response result is unexpectedly empty")]
+    EmptyResult { status: String, message: String },
```

Comment:
Should the error message include `status` or `message`?

Reply:
Oh yeah, let's include both! the status should only be a number so it's short. The message is what actually has the error

Reply:
done on https://github.com/gakonst/ethers-rs/pull/2426/commits/fa5c243a40adbd4af7aa291cb90ea5f52594ae9b